code/parachain/frame/cosmwasm/cli/src/error.rs

```diff
@@ -0,0 +1,23 @@
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+	#[error("Operation needs to be signed.")]
+	OperationNeedsToBeSigned,
+
+	#[error("Invalid funds. Format should be `asset_id-1:amount-1,asset_id-2:amount-2`")]
+	InvalidFundsFormat,
+
+	#[error("Invalid address.")]
+	InvalidAddress,
+
+	#[error("Invalid seed.")]
+	InvalidSeed,
+
+	#[error("Invalid phrase.")]
+	InvalidPhrase,
+
+	#[error("Shell command failure.")]
+	ShellCommandFailure,
+
+	#[error("{0}")]
+	ToolNotInstalled(String),
```

Comment:
which tool isn't installed?

Reply:
This is original code from CCW tool (see separate commit). 100% sure it needs to be improved. But that is not prio.